### PR TITLE
Clean up w2d styles and actionHref logic

### PR DESCRIPTION
**Pull request by bearfriend**
Somewhat dependent on https://github.com/BrightspaceUI/core/pull/1076

Refactors the way links and icons are styled, `actionHref` is set, and restores the link background color on hover.

### CODE REVIEW

**Review by mbardakovD2L — APPROVED** [review]
Looks good to me provided BrightspaceUI/core#1076 gets merged first

File: components/d2l-work-to-do/d2l-work-to-do-activity-list-item-basic.js [hunk 1]
```diff
@@ -200,32 +184,6 @@ class ActivityListItemBasic extends ListItemLinkMixin(SkeletonMixin(EntityMixinL
 		});
 	}
 
-	set actionHref(href) {  // This is a hack - Garbage setter function since list-mixin initializes value
-		this.requestUpdate('actionHref', this.evaluateAllHref ? this.evaluateAllHref : href);
-	}
-
-	/** Link to activity instance for user navigation to complete/work on activity */
-	get actionHref() {
```
Comment (JasperAtD2L) [inline components/d2l-work-to-do/d2l-work-to-do-activity-list-item-basic.js:208]: We still have to handle the skeleton. The skeleton attribute can still be applied even if the activity is fully loaded.

Comment (bearfriend) [reply components/d2l-work-to-do/d2l-work-to-do-activity-list-item-basic.js:208]: I handled that in CSS at the list-item level so as to uncomplicate things.

Comment (JasperAtD2L) [reply components/d2l-work-to-do/d2l-work-to-do-activity-list-item-basic.js:208]: Ah gotcha, so the list item mixin disables the link when skeleton attribute is applied?

Comment (bearfriend) [reply components/d2l-work-to-do/d2l-work-to-do-activity-list-item-basic.js:208]: Yep!

**Review by JasperAtD2L — APPROVED** [review]

**Pull request merged**